PostgreSQL.YAML-tmLanguage

```diff
@@ -83,10 +83,13 @@ repository:
 
   dollar_quotes:
     patterns:
-    - comment: "Assume multiline dollar quote is SQL; start if quote is at the end of the line."
+    - comment: "Assume multiline dollar quote is SQL body!
+        Start if double dollar quote is followed by no dollor quote till line ending.
```

Comment:
Last "dollor" typo left

Reply:
done

Reply:
wow, you are fast ))

Reply:
Now you have showed me the yaml ill investigate a bit further. Thus I work on repo at the moment